File: doc/src/codes/Users_Manual/modulation_2.rs
# extern crate autd3;
# extern crate tokio;
# use autd3::prelude::*;
# #[allow(unused_variables)]
# #[tokio::main]
# async fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::builder().open_with(autd3::link::Nop::builder()).await?;
autd.send(ConfigureModDelay::new(|dev, tr| {
    if dev.idx() == 0 && tr.idx() == 0 {
        1
    } else {
        0
    }
})).await?;
# Ok(())
# }

File: doc/src/codes/Users_Manual/modulation_2.cpp
#include "autd3.hpp"

autd.send(autd3::ConfigureModDelay([](const autd3::Device& dev,
                                      const autd3::Transducer& tr) {
  return dev.idx() == 0 && tr.idx() == 0 ? 1 : 0;
}));